sdv/data/got_families/character_families.csv

```diff
@@ -1,13 +0,0 @@
-character_id,family_id,generation,type
```

Comment:
What are all of these files for?

Reply:
I belive that those where datasets used for integration tests at some point (2 years ago). And those reside in the `data` , but since `.gitignore` is ignoring them you never saw them. 
Here is the commit https://github.com/sdv-dev/SDV/commit/9753a217955bfc222d925b06bef2ab5cdbf68d4a